src/storage-types/src/sources/load_generator.rs

```diff
@@ -447,8 +472,68 @@ impl RustType<ProtoLoadGeneratorSourceConnection> for LoadGeneratorSourceConnect
                     count_clerk,
                 },
                 Kind::Datums(()) => LoadGenerator::Datums,
+                Kind::KeyValue(kv) => LoadGenerator::KeyValue(kv.into_rust()?),
             },
             tick_micros: proto.tick_micros,
         })
     }
 }
+
+#[derive(Arbitrary, Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Default)]
+pub struct KeyValueLoadGenerator {
+    // The keyspace of the source.
```

Comment:
All of these should be doc comments
```suggestion
    /// The keyspace of the source.
```